### PR TITLE
DOC-2409: update 6 docs support page to include 6.8 row and End of Community Support column

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2409

Site: [Staging branch](http://docs-hotfix-6-doc-2409.staging.tiny.cloud/docs/tinymce/6/support)

Explanation: the new dates added here are the same as currently available on [7 Docs - Support Page](https://www.tiny.cloud/docs/tinymce/latest/support/) up to and including v6.8.

Changes:
* DOC-2409: update 6 docs support page to include 6.8 row and End of Community Support column

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [-] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [-] Files has been included where required `(if applicable)`
- [-] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [-] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/misc/supported-versions.adoc [hunk 1]
```diff
@@ -3,17 +3,18 @@
...
-|6.7 |2023-09-13 |2025-03-13
-|6.6 |2023-07-19 |2025-01-19
-|6.5 |2023-06-21 |2024-12-21
-|6.4 |2023-03-29 |2024-09-29
-|6.3 |2022-12-08 |2024-06-08
-|6.2 |2022-09-26 |2024-03-26
-|6.1 |2022-07-13 |2024-01-13
-|6.0 |2022-04-07 |2023-10-07
+|Version |Release Date |End of Community Support |End of Premium Support
+|6.8 |2023-12-06 |2024-10-31 |2025-06-06
```
Comment (kemister85) [inline modules/ROOT/partials/misc/supported-versions.adoc:9]: Once [this comment](https://github.com/tinymce/tinymce-docs/pull/3284/files#r1591781603) has been verified, we need to update this to match.
```suggestion
|6.8 |2023-12-06 |2024-03-20 |2025-06-06
```

Comment (kemister85) [reply modules/ROOT/partials/misc/supported-versions.adoc:9]: This has been confirmed.

Comment (FarzadHayat) [reply modules/ROOT/partials/misc/supported-versions.adoc]: thanks karl 👍

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Review by spocke — APPROVED** [review]

**Review by kemister85 — APPROVED** [review]

**Pull request merged**